packages/admin/README.md

```diff
@@ -3,15 +3,19 @@
 
 > A package/CLI tool to interact with the Airnode contracts across chains
 
+> This README is in sync with the master branch. You can find the **pre-alpha** documentation [here](https://github.com/api3dao/airnode/blob/pre-alpha/packages/admin/README.md).
```

Comment:
Why? I'd say it's actually good to add it every airnode package README.md. It's also just "write once and forget" thing, not sure what you mean by maintaining.

Things can (and probably will) change and it's easy to forget about pre-alpha.

Reply:
Oh, maybe you mean maintenance after we have more than `pre-alpha` version or after we have moved to sem ver?

At that point the API should be more stable and we could just remove this notice. But I'd say it's nice to inform people that the project is heavily under development.

Reply:
It's a subjective thing. Since our scope is huge, I avoid everything that will require a TODO (because I'll be the one to have to keep track). Here, we want to pretend as if pre-alpha has never existed once we move onto v0.1.0 to prevent confusion. Then, references to pre-alpha will need to be scrubbed clean, so I don't want them added just to be removed 1-2 months later.

Reply:
Ok.